src/helpers/implement-libcore.js

```diff
@@ -1,7 +1,4 @@
 // @flow
-import implementLibcore from '@ledgerhq/live-common/lib/libcore/platforms/nodejs'
-
-implementLibcore({
-  lib: require('@ledgerhq/ledger-core'),
-  dbPath: process.env.LEDGER_LIVE_SQLITE_PATH,
+process.send({
```

Comment:
semantic of this have changed and makes it a bit weird to understand. can you comment to explain how this works? might work a ascii schema to explain how things works between process

Reply:
Flow chart added as a comment to this file and in the PR description